Sources/Network/HTTPS/HttpProtocolConnectionFactory.swift

```diff
@@ -15,7 +15,7 @@ class HttpProtocolConnectionFactory: ProtocolConnectionFactory {
         config: AttestedConnectionConfig<ConsensusUrl>,
         targetQueue: DispatchQueue?,
         rng: (@convention(c) (UnsafeMutableRawPointer?) -> UInt64)?,
-        rngContext: Any?
+        rngContext: Any? = nil
```

Comment:
is this a temporary change ?

Reply:
Optional values default to nil regardless...  However, I believe I had to add in the setting to nil explicitly for some reason... will re-check this to see, but I ran into a similar thing just now when consolidating the MobileCoinClient.prepare* calls... where when the optional is not explicitly set, the compiler was having trouble identifying things, and seemed be trying to match up the params with the deprecated calls (i.e. it was complaining that Amount was not the expected type of UInt64)

Reply:
optional values do not default to nil, but nil is an acceptable value for the parameter. My comment is basically "we should force call-sites to be explicit about what they want the value to be, if they want nil they can pass nil."

Reply:
Ah, yes, you're correct, I think I was left over from my initial pass at wiring things up... not sure, but it's definitely not necessary to default to nil.